sysconfig/marconi/config.yaml

```diff
@@ -0,0 +1,10 @@
+config:
```

Comment:
Here it uses `SOFTS_DIR_PATH` whereas `APPS_DIR_PATH` is used for bb5.
Do you mind having a consistent variable for every machine? That would help the writing of deployment scripts.

Furthermore in bb5 config, there is a `tools` directory I don't find here. IMHO it should be consistent as well. Maybe bb5 is an exception.

For deployment of several machines using the same GPFS, it requires `SOFTS_DIR_PATH`. Instead would you consider putting the machine name inside config.yaml so that `SOFTS_DIR_PATH` remain the same in environment?



It would help me if the same of the machine was used in the config.yaml.

Reply:
Could you look at [this](https://github.com/BlueBrain/spack/tree/develop/sysconfig/bb5/users)?

APPS_DIR_PATH path if for core services tools. I think this should be compared with user configs on bb5?